src/app/App.js

```diff
@@ -40,9 +42,11 @@ const AppBody = ({ selectedLanguage, selectedTranslations, classes }) => (
     </Router>
   </IntlProvider>
 );
+
 AppBody.propTypes = {
   classes: PropTypes.object.isRequired
 };
+
```

Comment:
It seems like you do not use prettier.

Reply:
????

Reply:
pourquoi tu commit un fichier qui ne contiens que des espace ajoutées

Reply:
ça ne contient pas des espaces. Regarde bien, c'est prettier qui les a updatés